packages/vm/test/api/EIPs/eip-4895-BeaconChainWithdrawals.spec.ts

```diff
@@ -0,0 +1,112 @@
+import { Block } from '@ethereumjs/block'
+import { Chain, Common, Hardfork } from '@ethereumjs/common'
+import { FeeMarketEIP1559Transaction } from '@ethereumjs/tx'
+import { Address, zeros } from '@ethereumjs/util'
+import * as tape from 'tape'
+
+import { VM } from '../../../src/vm'
+
+import type { Withdrawal } from '@ethereumjs/block'
+
+const common = new Common({
+  chain: Chain.Mainnet,
+  hardfork: Hardfork.Merge,
+})
+
+const pkey = Buffer.from('20'.repeat(32), 'hex')
+
+tape('EIP4895 tests', (t) => {
+  t.test('EIP4895: withdrawals execute as expected', async (st) => {
+    const vm = await VM.create({ common })
+    vm._common.setEIPs([4895])
```

Comment:
This should be set using the`eips` constructor option along `Common` initialization and not by injecting this afterwards to the semi-private `_common` object.